<launch>
    <arg name="grab_param" default="grab_pose"/>
    
    <node name="visual_grab" type="visual_grab.py" pkg="piper_pick" output="screen">
                <param name="grab_param" value="$(arg grab_param)" />
    </node>
</launch>